### DISCUSSION TITLE
osu!direct Browse causes a crash before logging in

### DISCUSSION THREAD

**Discussion by sivadnivek** [ppy/osu#15860] [Q&A]
Hello,

If I press ctrl-d or select browse before logging in, cursor lags severely and lazer eventually crashes.
I didn't see anything in the logs, as they look to be generated after login, but I will attach.

[database.log](https://github.com/ppy/osu/files/7619049/database.log)
[network.log](https://github.com/ppy/osu/files/7619050/network.log)
[performance.log](https://github.com/ppy/osu/files/7619051/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7619052/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7619053/updater.log)

I don't know if this is a severe issue, it's of course circumvented by just logging in first.
Thanks for all of your hard work.
Best regards,
Kevin

**Comment by bdach** [comment]
I cannot reproduce this locally and the log looks clean. Is there any error in the system event viewer?

To open the event viewer, please type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>. The *Event viewer* window will appear.

Expand *Windows Logs* on the left and choose *Application*; it might take a while to load. Then please see if there are any errors that look related.

**Comment by sivadnivek** [comment]
`- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name=".NET Runtime" /> 
  <EventID Qualifiers="0">1023</EventID> 
  <Version>0</Version> 
  <Level>2</Level> 
  <Task>0</Task> 
  <Opcode>0</Opcode> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2021-11-29T21:41:11.3460000Z" /> 
  <EventRecordID>3686</EventRecordID> 
  <Correlation /> 
  <Execution ProcessID="0" ThreadID="0" /> 
  <Channel>Application</Channel> 
  <Computer>si7v3</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data>Description: A .NET application failed. Application: osu!.exe Path: C:\Users\siv\AppData\Local\osulazer\app-2021.1127.0\osu!.exe Message: Cannot use file stream for [C:\Users\siv\AppData\Local\osulazer\app-2021.1127.0\osu!.deps.json]: No such file or directory</Data> 
  </EventData>
  </Event>`

**Reply by sivadnivek** [reply]
Looks like just a missing json file. Is this created upon login?

**Reply by bdach** [reply]
no, that just looks like a pretty broken installation. i would recommend trying to reinstall the game and trying again.

**Reply by sivadnivek** [reply]
oh jeez lol. Ok

**Reply by sivadnivek** [reply]
I re-installed, got the same thing https://gape.name/20211129-crash.mkv

**Reply by sivadnivek** [reply]
I mean, oh well.. the game runs fine after logging in. This doesn't really affect gameplay. I just thought it might be report-worthy, it seems not to be
